first-analysis-steps/loki-functors.md

```diff
@@ -125,13 +137,31 @@ pv_finder_tool = appMgr.toolsvc().create(
 )
 pvs = evt['/Event/Rec/Vertex/Primary']
 best_pv = pv_finder_tool.relatedPV(cand, pvs)
+```
+Now, we can get the cosine of the direction angle for the candidate given the primary vertex: 
+```python
 from LoKiPhys.decorators import DIRA
 print DIRA(best_pv)(cand)
 ```
 
 Given that this is a very common operation, we have the possibility of using, in the context of a `DaVinci` application (Stripping, for example), a special set of functors, starting with the `BPV` prefix (for Best PV), which will get the PV for us.
 Some functors also end with the suffix `DV`, which means they can only be used in the `DaVinci` context.
 
+To get the quality of impact parameter of the candidate, one needs as well to call a distance calculator:
+```python
+from GaudiPython.Bindings import AppMgr, gbl 
+gaudi = AppMgr() 
+distCal = gaudi.toolSvc().create("LoKi::DistanceCalculator", interface = gbl.IDistanceCalculator) 
```

Comment:
From the [python style guide](https://www.python.org/dev/peps/pep-0008/#other-recommendations):
> Don't use spaces around the = sign when used to indicate a keyword argument or a default parameter value.